proknow-sdk/Patient/Entities/EntityScorecards.cs

```diff
@@ -16,28 +16,28 @@ public class EntityScorecards
     {
         private readonly ProKnowApi _proKnow;
         private readonly string _workspaceId;
-        private readonly string _entityId;
+        private readonly string _patientId;
```

Comment:
Same story as with previous comment. This class need to be moved up and renamed.